### DISCUSSION TITLE
windows version

### DISCUSSION THREAD

**Discussion by alphaleadership** [searxng/searxng#4029] [Q&A] [answered]
windows version exist

**Comment by unixfox** [comment]
you can use WSL to run in it inside a linux environment: https://learn.microsoft.com/en-us/windows/wsl/install

**Reply by alphaleadership** [reply]
Add Windows full compatibilty Is hard

**Comment by return42** [comment, marked as answer]
We do not really support M$ Windows natively .. our main focus is on Linux/Unix .. If native support does not work on the desired OS, you should use the docker image.

**Reply by alphaleadership** [reply]
the docker image dont have good windows support
